.github/workflows/build.yml

```diff
@@ -364,25 +312,41 @@ jobs:
 
       - name: Build Docker image
         uses: docker/build-push-action@1104d471370f9806843c095c1db02b5a90c5f8b6 # v3.3.1
+        env:
+          IMAGE_TAG: ${{ steps.tag.outputs.image_tag }}
+          PROJECT_NAME: ${{ steps.setup.outputs.project_name }}
         with:
           context: ${{ matrix.base_path }}/${{ matrix.project_name }}
           file: ${{ matrix.base_path }}/${{ matrix.project_name }}/Dockerfile
           platforms: linux/amd64
           push: true
-          tags: ${{ steps.tag-list.outputs.tags }}
+          tags: ${{ env.AZ_REGISTRY }}/${{ env.PROJECT_NAME }}:${{ env.IMAGE_TAG }}"
```

Comment:
```suggestion
          tags: ${{ env.AZ_REGISTRY }}/${{ env.PROJECT_NAME }}:${{ env.IMAGE_TAG }}
```
⚠️ Extra double quote included, this can be removed.